lte/gateway/release/agw_setup_ubuntu.sh

```diff
@@ -0,0 +1,36 @@
+
+
+apt update
+
+# rename interfaces
```

Comment:
Before renaming all interfaces let's make sure they are not already setup properly